plugins/cost-insights/src/components/CostInsightsPage/CostInsightsPage.tsx

```diff
@@ -165,6 +174,43 @@ const CostInsightsPage = () => {
     );
   }
 
+  const onProjectSelect = (project: Maybe<string>) => {
+    setProject(project);
+    dispatchLoadingReset(loadingActions);
+  };
+
+  const CostOverviewBanner = () => (
+    <Box
+      px={3}
+      marginTop={10}
+      display="flex"
+      flexDirection="row"
+      justifyContent="space-between"
+    >
+      <Box minHeight={40} width="75%">
+        <Typography variant="h4" style={{ paddingTop: 10 }}>
```

Comment:
Nitpick but we should generally try to avoid using the style prop so we can keep specificity low.

Could the `pt` prop on parent `Box` be used here?